phpunit.xml

```diff
@@ -2,6 +2,8 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="phpunit.xsd"
          bootstrap="tests/bootstrap.php"
+         cacheResult="true"
+         executionOrderDefects="defectsFirst"
```

Comment:
I would prefer not to change the configuration of PHPUnit's own configuration (for now, at least).

Reply:
Please elaborate @sebastianbergmann ,

While I can agree that setting `ignore-dependencies` can bring failures for codebase with hidden dependencies in tests (which, anyway, is great to detect and devs shall fix that in their repos!),
having caching by default is great, running test that failed on previous run as first will save a lot of time to the devs!
And i'm afraid that if it won't be the default, very few of us in the wild will be even aware of this option...

Reply:
Oops. This actually was an accidental commit-push-zzzzz from me yesterday evening, but these are the standard settings to speed up development.

Most developers would never find out about the option, I have to agree with @keradus. Not that they would not _want_ it, yet many end users simply never read the changelog and docs in that amount of detail. For context, I still see test suites that do not even use fixtures or dataproviders, handling those things themselves manually.